components/Layout/AudioControls.vue

```diff
@@ -0,0 +1,222 @@
+      <v-col cols="3" class="d-flex d-md-none px-0 align-center justify-end">
+        <v-btn icon>
+          <v-icon>mdi-heart</v-icon>
+        </v-btn>
+        <v-btn icon @click="togglePause">
+          <v-icon>
+            {{ isPaused ? 'mdi-play' : 'mdi-pause' }}
+          </v-icon>
+        </v-btn>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { ImageType } from '@jellyfin/client-axios';
+import Vue from 'vue';
+import { mapActions, mapGetters } from 'vuex';
+import imageHelper from '~/mixins/imageHelper';
+import timeUtils from '~/mixins/timeUtils';
+import { PlaybackStatus } from '~/store/playbackManager';
+
+export default Vue.extend({
+  mixins: [timeUtils, imageHelper],
+  data() {
+    return {
+      clicked: false,
+      currentInput: 0
+    };
+  },
+  computed: {
+    ...mapGetters('playbackManager', ['getCurrentItem']),
+    runtime(): number {
+      return this.ticksToMs(this.getCurrentItem.RunTimeTicks) / 1000;
+    },
+    isPaused(): boolean {
+      return this.$store.state.playbackManager.status === PlaybackStatus.paused;
+    },
+    sliderValue: {
+      get(): number {
+        if (!this.clicked) {
+          return this.$store.state.playbackManager.currentTime;
+        }
+        return this.currentInput;
+      }
+    },
+    realPosition: {
+      get(): number {
+        return this.$store.state.playbackManager.currentTime;
+      }
+    }
+  },
+  methods: {
+    ...mapActions('playbackManager', [
+      'changeCurrentTime',
+      'setLastItemIndex',
+      'resetCurrentItemIndex',
+      'setNextTrack',
+      'setPreviousTrack',
+      'unpause',
+      'pause'
+    ]),
+    getImageUrl(itemId: string): string {
+      const element = this.$refs.albumCover as HTMLElement;
+      return this.getImageUrlForElement(ImageType.Primary, {
+        itemId,
+        element
+      });
+    },
+    getRuntime(seconds: number): string {
+      const minutes = Math.floor(seconds / 60);
+      seconds = Math.floor(seconds - minutes * 60);
+
+      /**
+       * Formats the second number
+       * E.g. 7 -> 07
+       *
+       * @param {string} seconds Number to format
+       * @returns {string} Formatted seconds number
+       */
+      function formatSeconds(seconds: string): string {
+        return ('0' + seconds).slice(-2);
+      }
+
+      return `${minutes}:${formatSeconds(seconds.toString())}`;
+    },
+    onPositionChange(value: number) {
+      if (!this.clicked) {
+        this.changeCurrentTime({ time: value });
+      }
+    },
+    onInputChange(value: number) {
+      this.currentInput = value;
+    },
+    onClick() {
+      this.currentInput = this.realPosition;
+      this.clicked = !this.clicked;
+    },
+    stopPlayback() {
+      this.setLastItemIndex();
+      this.resetCurrentItemIndex();
+      this.setNextTrack();
+    },
+    togglePause() {
+      if (this.isPaused) {
+        this.unpause();
+      } else {
+        this.pause();
```

Comment:
```suggestion
    onPositionChange(value: number): void {
      if (!this.clicked) {
        this.changeCurrentTime({ time: value });
      }
    },
    onInputChange(value: number): void {
      this.currentInput = value;
    },
    onClick(): void {
      this.currentInput = this.realPosition;
      this.clicked = !this.clicked;
    },
    stopPlayback(): void {
      this.setLastItemIndex();
      this.resetCurrentItemIndex();
      this.setNextTrack();
    },
    togglePause(): void {
      if (this.isPaused) {
        this.unpause();
      } else {
        this.pause();
```

Sorry, I missed this in my commit and we need it if we want to merge #466 😅